scripts/clean-workingdir.sh

```diff
@@ -6,6 +6,6 @@ WORKING_DIR=$(cd `dirname $0`/.. && pwd)
 
 cd $WORKING_DIR
 
-rm -rf agent/config agent/pipelines server/pipelines server/db/config.git
+rm -rf server/test-repo agent/config agent/pipelines server/pipelines server/db/config.git
```

Comment:
Don't mind this -- it's just a script I use for cleaning my dev environment without killing my IDEA settings. Didn't think it was worth a separate PR, so I'm smuggling this one in here :)